install.sh

```diff
@@ -72,9 +72,14 @@ if [ -d "/etc/systemd/system" ]; then
   fetch "$url" "$tmpDir/service" || oops "failed to download '$url'"
   if [ ! -f "/etc/systemd/system/${agent}" ]; then
     echo "Will use sudo to install systemd service, you will probably need to enter credentials"
-    ${sudo} cp -f "$tmpDir/service" /etc/systemd/system/${agent}.service
+    if [ -z ${apikey+x} ]; then
+      echo "Please enter API key: "
+      read apikey
```

Comment:
we can do oneline? 
`read -p "Please enter API key: " apikey`

Reply:
I would also add short description what API key should user specify. maybe `Please enter API key generated on bolt.observer website:` 🤔